### PR TITLE
download button for supporting data entry pages

### PR DESCRIPTION
**Pull request by aurel-l**
## Purpose
Add download button to supporting data entry pages.

## Approach
Create one reusable prop-less component getting the information it needs from the location (through `useRouteMatch`).
Starting to slowly refactor parts of the supporting data approach in order to handle them in a more common way.

## Testing
Tried to download some entries

## Checklist
- [x] My PR is scoped properly, and “does one thing only”
- [x] I have reviewed my own code
- [x] I have checked that linting checks pass and type safety is respected
- [ ] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why

### CODE REVIEW

File: src/supporting-data/database/config/download.ts [hunk 1]
```diff
@@ -2,5 +2,4 @@ import { FileFormat } from '../../../shared/types/resultsDownload';
 
 export const fileFormatsResultsDownload = [FileFormat.rdfXml, FileFormat.json];
```
Comment (xwatkins) [inline src/supporting-data/database/config/download.ts:3]: Noticed that the "Preview N" button in download doesn't work for RDF/XML - (error in the request syntax?)

File: src/supporting-data/shared/components/EntryDownload.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import { fileFormatEntryDownload as citationsFFED } from '../../citations/config/download';
+import { fileFormatEntryDownload as databaseFFED } from '../../database/config/download';
+import { fileFormatEntryDownload as diseasesFFED } from '../../diseases/config/download';
+import { fileFormatEntryDownload as keywordsFFED } from '../../keywords/config/download';
+import { fileFormatEntryDownload as locationsFFED } from '../../locations/config/download';
+import { fileFormatEntryDownload as taxonomyFFED } from '../../taxonomy/config/download';
+
+import { Namespace } from '../../../shared/types/namespaces';
+
+const lut = new Map([
```
Comment (xwatkins) [inline src/supporting-data/shared/components/EntryDownload.tsx:16]: Not sure what `lut` stands for?

**Review by xwatkins — CHANGES_REQUESTED** [review]
Small issue with RDF/XML and an obscure variable name but good approach. Will be useful for ID mapping download too I think...

File: src/supporting-data/shared/components/EntryDownload.tsx (hunk 2 above)
Comment (aurel-l) [reply src/supporting-data/shared/components/EntryDownload.tsx:16]: Look up table. I already changed it in the following branch about the map to buttons

File: src/supporting-data/database/config/download.ts (hunk 1 above)
Comment (aurel-l) [reply src/supporting-data/database/config/download.ts]: It's the issue I mentioned this morning. RDF/XML is not supported by the backend (at the moment). I have now removed it.

File: src/supporting-data/shared/components/EntryDownload.tsx (hunk 2 above)
Comment (aurel-l) [reply src/supporting-data/shared/components/EntryDownload.tsx:16]: Called `formatMap` in the other branch

Comment (xwatkins) [reply src/supporting-data/shared/components/EntryDownload.tsx:16]: I am familiar with LUTs in video editing but didn't think this was about adding a vintage Kodak look to the website 😝

**Review by xwatkins — APPROVED** [review]

**Pull request merged**